### PR TITLE
[Addon]Put Vector controller in loki addon

### PR DESCRIPTION
**Pull request by wangyikewxgm**
### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [ ] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [ ] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [ ] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Please be aware of this contribution rules when contribute addons.
 
 - An experimental addon must meet these conditions to be promoted as a verified one.
 
-  - This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
+  - This addon must be tested by addon's [e2e-test](./test/e2e-test/0b addon-test) to guarantee this addon can be enabled successfully.
```
Comment (Somefive) [inline README.md:41]: What is it?

File: addons/loki/definitions/collect-logs.cue [hunk 2]
```diff
@@ -3,7 +3,7 @@ import (
 	"path"
 )
 
-"collect-logs": {
+"log-collect-file": {
```
Comment (Somefive) [inline addons/loki/definitions/collect-logs.cue:6]: If change this name, also change the filename. BTW, `log-collect-file` is weird, suggest using `collect-file-logs` or `file-logs-collector`.

File: addons/loki/definitions/log-collect-stdout.cue [hunk 3]
```diff
@@ -0,0 +1,1474 @@
+// We put Definitions in definitions directory.
+// References:
+// - https://kubevela.net/docs/platform-engineers/cue/definition-edit
+// - https://kubevela.net/docs/platform-engineers/addon/intro#definitions-directoryoptional
+
+"log-collect-stdout": {
```
Comment (Somefive) [inline addons/loki/definitions/log-collect-stdout.cue:6]: This definition includes the dashboard for nginx/apache right?

**Comment by Somefive** [comment]
BTW, add changes to the addon version number.

File: addons/loki/definitions/log-collect-stdout.cue (hunk 3 above)
Comment (wangyikewxgm) [reply addons/loki/definitions/log-collect-stdout.cue]: only nginx's dashboard

File: addons/loki/definitions/collect-logs.cue (hunk 2 above)
Comment (wangyikewxgm) [reply addons/loki/definitions/collect-logs.cue]: good suggestion.

Comment (wangyikewxgm) [reply addons/loki/definitions/collect-logs.cue]: good suggestion.

File: README.md (hunk 1 above)
Comment (wangyikewxgm) [reply README.md]: fixed

**Review by Somefive — APPROVED** [review]

**Review by Somefive — DISMISSED** [review]

File: addons/loki/definitions/stdout-logs-collector.cue [hunk 4]
```diff
@@ -0,0 +1,1474 @@
...
+              .message = parse_apache_log!(.message, format: "common")
+              """
+						}
+						if parameter.parser != _|_ && parameter.parser == "nginx" {
+							source: """
+              .message = parse_nginx_log!(.message, "combined")
+              """
+						}
+						if parameter.parser != _|_ &&
+					}
```
Comment (Somefive) [inline addons/loki/definitions/stdout-logs-collector.cue:67]: Unfinished code?

**Review by Somefive — APPROVED** [review]

**Review by wonderflow — APPROVED** [review]

**Pull request merged**